config/routes.rb

```diff
@@ -19,7 +19,16 @@
       get :rejected, :closed
     end
   end
-  resources :organizations
+
+  resources :organizations, only: [:index, :new, :edit, :update, :create, :destroy] do
+    collection do
+      get :deleted
+    end
+
+    member do
+      get :restore
```

Comment:
This shouldn't be a get method. Post, put, or patch would be ideal (I favor post for custom routes for simplicity).